README.md

```diff
@@ -7,24 +7,26 @@
 ## Get started
 
 1. `npm install --save @daostack/access_control`
-2.  Use it in your smart contract
-    ```javascript
-    import "@daostack/access_control/contracts/Protected.sol";
+2. Use it in your smart contract
 
-    contract Foo is Protected {
-        // Foo stuff...
-    }
-    ```
+   ```javascript
+   import "@daostack/access_control/contracts/Protected.sol";
+
+   contract Foo is Protected {
+       // Foo stuff...
+   }
+   ```
 
 Check out the [documentation](https://daostack.github.io/access_control/) for more!
 
 ## Concept
+
 The basic idea is that there are a set of "locks"/"topics" each with a set of "keys". A contract can protects one of it's methods with a lock (or more) such that anyone with the appropriate key can use it to access this method.
-Each key can have an expiration time, # of uses. Accounts can also transfer some (or all) of their capabilities to other accounts if allowed. Initially the contract itself has a "master key" (infinite uses, no expiration), and then it can pass some partial capabilities to the desired accounts.
+Each key can have a start time, expiration time, # of uses. Accounts can also transfer some (or all) of their capabilities to other accounts if allowed. Initially the contract itself has a "master key" (infinite uses, no expiration, no start time limitation), and then it can pass some partial capabilities to the desired accounts.
 
 Features:
 
-- Can define a number of uses & expiration date on keys and if to enable the key to be transfrred.
+- Can define a number of uses & expiration date & start time on keys and if to enable the key to be transfrred.
```

Comment:
A key can be configured with the following charachteristics : 
number of uses , expiration date ,start time , transferrable .